src/main/render/values.ts

```diff
@@ -139,3 +146,11 @@ function renderList(fieldType: ListType, node: ConstList): ts.ArrayLiteralExpres
 
   return ts.createArrayLiteral(values)
 }
+
+function renderBuffer(node: StringLiteral): ts.NewExpression {
```

Comment:
Thanks for doing this. This should probably be created using Buffer.from as new Buffer(sting) is deprecated.

Reply:
No problem! Woops, forgot about that. I'll make that change right now.